gallery/src/pages/Home.tsx

```diff
@@ -24,35 +27,157 @@ const Home = () =>
-      </NxP>
-      <NxH4>JavaScript &amp; TypeScript Utilities</NxH4>
-      <NxP>Utility function and datatypes available to users of RSC.</NxP>
+      <NxDescriptionList>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term><NxFontAwesomeIcon icon={faReact} fixedWidth /></NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Documentation for a React component or family of closely related React components. Typically includes
+            technical details about the props that may be passed to the component along with live examples of its use.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term><NxFontAwesomeIcon icon={faCode} fixedWidth /></NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Documentation for usage of bare CSS classes within HTML. Some RSC "components" consist almost entirely of
+            styling with little or no special behavior that must be implemented. For these cases, the bare classes
+            are themselves a documented and stable part of the RSC API, allowing them to be used even in applications
+            which do not use React. Many of these components are also exposed via React "convenience components"
+            which simply pair the class in question with the HTML tag to which it is most typically applied.
+            Convenience components are not typically documented separated and are simply mentioned on these non-React
+            pages. In some cases however, a component may have a simple form exposed as a base CSS class while
+            also having a non-trivial React component implementing a certain amount of behavior which a non-React
+            application would have to manage themselves. In these cases, the React component is documented on a separate
+            page from the base CSS classes. Live examples on these pages are often implemented in bare HTML, though
+            some are still implemented in React in order to simplify the inclusion of things such as icons.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term><NxFontAwesomeIcon icon={faRulerCombined} fixedWidth /></NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Complex, compound layout documentation. These pages feature combinations of RSC components that have
+            separately have their own documentation pages, but which must also be documented in unison in order to
+            understand the full effect. For instance, one such page demonstrates the use
+            of <NxCode>nx-read-only</NxCode>, <NxCode>nx-grid</NxCode>, and <NxCode>nx-tile</NxCode> together in order
+            to implement a common data display pattern.
```

Comment:
Suggestion: This section contains documentation on the combination usage of RSC components, when used together. It explains how these components can be used together to display a common pattern.